src/content/enterprise/administration/certificates.mdx

```diff
@@ -32,6 +35,9 @@ ingress-nginx:
       default-ssl-certificate: $(POD_NAMESPACE)/your-ssl-certificate-secret
 ```
 
+In order to generate your certificates with [Cert Manager](https://cert-manager.io/) you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
```

Comment:
@rberrelleza I would have this just in the community forum

Reply:
Those products don't use wildcard certificates. CM for wildcard certificates isn't the easiest option